### DISCUSSION TITLE
change and input events on input type="range" don't work on ios.

### DISCUSSION THREAD

**Discussion by metalalchemist** [apache/cordova#393] [Q&A]
hello, i'm making a input type=range that control the volume of an audio and firs i use the change event and then i used input.

but don't work. when i move the slider the music still are in the same volume.
the problem only is in ios. when i use in android works perfectly.
any ideas? thanks

**Comment by breautek** [comment]
Can you show us some sample code?

Also are using standard browser Audio elements or are you using a plugin to control audio playback?

**Comment by metalalchemist** [comment]
I'm use audio element navigator, audio tag html5
sure this is my code.
``` html
<label for="vmusica" id="smusica">Volumen de la música</label><input type="range" min="0" max="100" step="10" id="vmusica">
```

``` javascript
document.querySelector("#vmusica").addEventListener("change", function () {
var audio = document.querySelector("#vmusica");
audio.volume=document.querySelector("#vmusica").value/100);
});
```

i tested it in android webview and in my pc withouth cordova in Microsoft Edge and in Safary.
I change the changeEvent for inputEvent to but don't work.
